packages/gatekeeper/src/helpers/services.ts

```diff
@@ -94,15 +99,14 @@ export async function fetchUser(
 }
 
 /** some docstring */
-export const fetchState = async (
-  url: string,
-  authenticateActionCreator: ActionCreator<AuthenticateAction> = authenticateUser,
-  recordResultActionCreator: ActionCreator<RecordAction> = recordResult,
-  authenticationProgressCreator: ActionCreator<
-    AuthenticationProgressAction
-  > = authenticationProgress,
-  errorCallbackFn: ErrorCallback = errorCallback
-) => {
+export const fetchState = async ({
+  url = '',
```

Comment:
Can we make this function not have a default for `url`?  I want the Typescript compiler to complain when this function is called without a URL (coz it does not make sense to do so).

Reply:
done